packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -352,9 +352,9 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
    * Close menu callout if it is open
    */
   @autobind
-  public dismissMenu(): void {
+  public dismissMenu(dismissMenu = true): void {
```

Comment:
I think you want the default to be false. It is a more predictable behavior. Also you need some better description of what the flag is intended to do. It doesn't really make sense to call `dismissMenu(false)` and expect it to stay open...